ts/utils/clipboard.ts

```diff
@@ -3,6 +3,9 @@ import { Clipboard } from "react-native";
 
 import I18n from "../i18n";
 
+/**
+ * Copy a text to the device clipboard and optionally give a feedback.
+ */
 export const copyToClipboard = (
   text: string,
   withFeedback: boolean = false
```

Comment:
I don't see the need for this since calling this with `false` is equivalent to calling `Clipboard.setString`